src/app/Main/Body/QueryPage/Drawer/Displayed_Features/DisplFeatures-Selector.component.tsx

```diff
@@ -6,7 +6,6 @@ import _ from "lodash";
 import { SelectorComponent } from "../../../../../Shared/Selector.component";
 import { FeatureType } from "../../../../../Shared/Context/DataContext";
 import { FilterType } from "../../../../../Shared/Model/Filter.model";
-import { generateFeatureList } from "./generateFeatureList.service";

```

Comment:
Good, this service was kind of over done. In general services are for doing more complicated algorithms, things that are shared between components or wrapping frameworks and external functionalities.